src/domain-services/flows/flow-service.ts

```diff
@@ -473,91 +485,88 @@ export class FlowService {
       distinct: ['objectID', 'versionID'],
     });
 
+    // Build list of parent IDs from categoryRefs
+    const parentIDs = categoryRefs.map((ref) =>
+      createBrandedValue(ref.objectID)
+    );
```

Comment:
Brand the ID as `FlowId`. Method `createBrandedValue()` is generic and you can use it as `createBrandedValue<number, FlowId>()`, but shorter way is this:
```suggestion
    const parentIDs: FlowId[] = categoryRefs.map((ref) =>
      createBrandedValue(ref.objectID)
    );
```